.travis.yml

```diff
@@ -28,7 +28,6 @@ before_script:
   - sleep 3 # give xvfb some time to start
 
 script:
-  - npm run check-package
```

Comment:
Why remove these checks?

Reply:
This doesn't answer my question, and more importantly, I just verified and it does not perform the check to ensure versions or names are different like this package check does.

We need to bring this check back, as you can see:

```sh
js-dpp nmarley$ npm run check-package:version

> @dashevo/dpp@0.4.5 check-package:version /Users/nmarley/projects/js-dpp
> test $(jq -r .version package.json) = $(jq -r .version package-lock.json)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @dashevo/dpp@0.4.5 check-package:version: `test $(jq -r .version package.json) = $(jq -r .version package-lock.json)`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @dashevo/dpp@0.4.5 check-package:version script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nmarley/.npm/_logs/2019-02-06T14_24_36_687Z-debug.log
js-dpp nmarley$ npm ci --quiet
npm WARN prepare removing existing node_modules/ before installation

> fsevents@1.2.4 install /Users/nmarley/projects/js-dpp/node_modules/fsevents
> node install

[fsevents] Success: "/Users/nmarley/projects/js-dpp/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile
added 861 packages in 6.968s
js-dpp nmarley$ echo $?
0
```

Reply:
Oh... my bad, sorry! I wrongly interpreted documentation.